docker/Dockerfile

```diff
@@ -1,22 +1,16 @@
 FROM python:3.7-slim AS build-image
 LABEL maintainer="graphenex.project@protonmail.com"
 ENV LC_ALL=C.UTF-8
-
-RUN apt-get update
-RUN apt-get install -y --no-install-recommends build-essential gcc curl
-
-RUN echo "**** Fetching the repository ****" && \
-    VERSION=$(curl --silent "https://api.github.com/repos/graphenex/grapheneX/releases/latest" | \
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends build-essential gcc curl
+RUN VERSION=$(curl --silent "https://api.github.com/repos/graphenex/grapheneX/releases/latest" | \
     grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/') && \
```

Comment:
Instead of this mess, using an environment variable would be better IMO. Such as 
```docker
ENV VERSION 1.0.0
```

Reply:
I think we should pull the latest version from GitHub since we're not copying the source files into the image. It's easier to build the latest version this way.

Reply:
What I mean is, instead of parsing the version using SED, give the version as a parameter. With that approach, you can increase the version when you have new one. Not saying copy the files from one place to another. You still need to pull "a version" you're specifying with the param.

Reply:
With or without the parameter, the user have to deal with the 'mess' you referred for pulling the latest tag from GitHub unless if s/he is okay with checking the latest version on every release. So I suggest that we should keep this inside the image build. Moving this command to outside won't affect the purpose of building the latest version.
But if you're suggesting this for the flexibility of providing other versions than just the latest, then maybe you're right. I didn't say anything about it since I'm thinking of using tags on Docker Hub.